google/cloud/sql/connector/InstanceConnectionManager.py

```diff
@@ -364,6 +357,7 @@ def _update_current(self, future: concurrent.futures.Future) -> None:
         logger.debug("Entered _update_current")
         with self._lock:
             self._current = future
+            # Ephemeral certificate expires in 1 hour, so we schedule a refresh to happen in 55 minutes.
             self._next = self._loop.create_task(self._schedule_refresh(55 * 60))
```

Comment:
It may make sense to pull this into a private constant to avoid the inline magic number